Contuity/Models/FollowUp.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  FollowUp.swift
+//  Contuity
+//
+//  Created by Lexi Spikerman on 10/2/18.
+//  Copyright © 2018 Generate. All rights reserved.
+//
+
+import Foundation
+
+class FollowUp {
+    
+    var id: Int
+    var jot_id: Int
+    var text: String?
+    var datetime: NSDate?
+    
+    init(id: Int, jot_id: Int, text: String?, datetime: NSDate?){
```

Comment:
I think if you convert the model to a struct then you get this initializer for free and don't need to explicitly add it in the code